src/chainstate/stacks/db/blocks.rs

```diff
@@ -1092,6 +1093,20 @@ impl StacksChainState {
         Ok(Some(microblocks))
     }
 
+    /// stacks_block _must_ have been committed, or this will return an error
+    pub fn get_parent(&self, stacks_block: &StacksBlockId) -> Result<StacksBlockId, Error> {
```

Comment:
This method scares me with its low-level access to the MARF.  Why not just update the Stacks header DB to store the stacks block ID for its parent?

Reply:
Good idea -- I think that may be useful elsewhere as well. Updated in 72514b889